src/foam/nanos/dig/DIGOwnerRuleAction.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Is this needed, can we not use 'createdBy' as that has both createdBy and createdByAgent.